### ISSUE TITLE
Scheduled Campaign but Email was not sent

### ISSUE THREAD

**Issue by mirkatson** [gophish/gophish#1150]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: 
0.6

Brief description of the issue: 
I've scheduled a campaign however when the time is up, the email wasnt sent out. It changed from Scheduled to "Error". Test email from Sending profile is fine, no error.

What are you expecting to see happen? :
Email being sent out. 

What are you seeing happen?
Email wasnt sent out.

Thanks Jordan!

**Comment by mirkatson** [comment]
I did restarted the server too. Also ensuring Postfix and Gophish service are running.

**Comment by S0larflare** [comment]
What does the error say? What does the console say when the error occurred?

Sent from my iPhone

On 24 Jul 2018, at 16:37, mirkatson <notifications@github.com<mailto:notifications@github.com>> wrote:


I did restarted the server too. Also ensuring Postfix and Gophish service are running.

**Comment by mirkatson** [comment]
@S0larflare  there wasnt any error from daemon.log or mail.log though

**Comment by mirkatson** [comment]
@S0larflare  One of the line in mail.log was:

gophish[1183]: time="2018-07-24T16:02:14Z" level=warning msg="record not found"

**Comment by mirkatson** [comment]
I have checked the maillogs table. as well. Their send_date are OK, but they are not being sent.

Is there outstanding tasks GoPhish needs to complete first?
